science/orthanc/Makefile

```diff
@@ -0,0 +1,57 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc
+DISTVERSION=	1.9.3
+CATEGORIES=	science
+MASTER_SITES=	https://www.orthanc-server.com/downloads/get.php?path=/orthanc/
+DISTNAME=	Orthanc-${PORTVERSION}
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Lightweight DICOM server for healthcare and medical research
+
+LICENSE=	GPLv3
+LICENSE_FILE=	${WRKSRC}/COPYING
+
+LIB_DEPENDS=	libboost_atomic.so:devel/boost-libs \
+		libcivetweb.so:www/civetweb \
+		libcurl.so:ftp/curl \
+		libdcmtkcharls.so:graphics/dcmtk \
+		libgtest.so:devel/googletest \
+		libicuuc.so:devel/icu \
+		libjsoncpp.so:devel/jsoncpp \
+		libpng16.so:graphics/png \
+		libpugixml.so:textproc/pugixml \
+		libtiff.so:graphics/tiff \
+		libuuid.so:misc/e2fsprogs-libuuid
+
+USES=		cmake gnome iconv jpeg lua python:build sqlite ssl
+USE_GNOME=	libxml2
+USE_LDCONFIG=	yes
+
+WRKSRC=		${WRKDIR}/Orthanc-${PORTVERSION}
+
+CMAKE_SOURCE_PATH=	${WRKSRC}/OrthancServer
+CMAKE_OFF=	BUILD_CONNECTIVITY_CHECKS UNIT_TESTS_WITH_HTTP_CONNEXIONS
+CMAKE_ON=	USE_SYSTEM_BOOST USE_SYSTEM_CIVETWEB USE_SYSTEM_DCMTK \
+		USE_SYSTEM_GOOGLE_TEST USE_SYSTEM_JSONCPP USE_SYSTEM_LIBICONV \
+		USE_SYSTEM_LIBJPEG USE_SYSTEM_LIBPNG USE_SYSTEM_LUA \
+		USE_SYSTEM_OPENSSL USE_SYSTEM_PUGIXML USE_SYSTEM_SQLITE \
+		USE_SYSTEM_UUID USE_SYSTEM_ZLIB
+CFLAGS+=	-DNDEBUG
+CXXFLAGS+=	-DNDEBUG
+
+USE_RC_SUBR=	orthanc
+
+USERS=		orthanc
+GROUPS=		orthanc
```

Comment:
Note that when defining this, we also need to update ports/{UIDs,GIDs}, but it seems got forgot when deleted. :)

Reply:
Oh! Today I learned. Though from what I can see, the UID/GID stayed the same, so it actually works as-is this time, correct?

Reply:
Yes, just letting you know this patch doesn't contain UIDs/GIDs is because of a previous fault. :)